src/omv/modules/py_csi.c

```diff
@@ -1028,6 +1028,15 @@ static mp_obj_t py_omv_csi_ioctl(size_t n_args, const mp_obj_t *args) {
             }
             break;
         }
+        case OMV_CSI_IOCTL_GENX320_SET_AFK: {
+            if (n_args == 2) {
+                error = omv_csi_ioctl(request, mp_obj_get_int(args[1]));
+            } else if (n_args == 4) {
+                error = omv_csi_ioctl(request, mp_obj_get_int(args[1]), mp_obj_get_int(args[2]),
+                                      mp_obj_get_int(args[3]));
+            }
```

Comment:
Add error check on args not being 2 or 4.

Reply:
No need, error is set to invalid args.